typeguards.ts

```diff
@@ -32,8 +33,46 @@ import {
   Rule,
   ScaleDenominator,
   StrMatchesFunctionOperator,
-  TextSymbolizer
-} from './style';
+  TextSymbolizer,
+  CategorizeFunctionFilter,
+  CategorizeFunctionOperator,
+  Expression,
+  FunctionCall,
+  LiteralValue,
+  PropertyName
+} from './index';
+import _ from 'lodash';
```

Comment:
This looks like an automated import of all of lodash. I guess this was not intended.

Reply:
Nice catch, removed in f25b142